### ISSUE TITLE
graph views: stacked attribute ignored

### ISSUE THREAD

**Issue by gmarcon** [odoo/odoo#5491]
Hello,

  I have a graph view over account.invoice for partner aging done like this:

```xml
<?xml version="1.0"?>
<graph string="Customer Aging" type="bar" stacked="True">
    <field name="date_due" type="row" />
    <field name="partner_id" type="row" />
    <field name="residual" type="measure"/>
</graph>
```

Unfortunately, the `stacked="True"` seems to be completely ignored. I am on version 8.0-d30103c.

Thank you and regards,
Giulio Marcon

**Comment by anilsavani** [comment]
in addons/web_graph/static/src/js/graph_view.js
find 
             stacked : (arch.attrs.stacked === 'True'),
add new line after it
            bar_ui : (arch.attrs.stacked === 'True'?"stack":"group"),

and that should work

**Comment by gmarcon** [comment]
Thank you, worked like a charm!

**Comment by ged-odoo** [comment]
issue fixed in 9.0, commit https://github.com/odoo/odoo/commit/f652d384a2703ed9860279b85bb8080cc6069021
